osf/management/commands/update_institution_project_counts.py

```diff
@@ -0,0 +1,52 @@
+import datetime as dt
+
+from django.core.management.base import BaseCommand
+
+from osf.metrics import InstitutionProjectCounts, UserInstitutionProjectCounts
+from osf.models import Institution, Node
+
+
+def update_institution_project_counts():
+    now = dt.datetime.now()
+
+    for institution in Institution.objects.all():
+
+        if institution.osfuser_set.exists() and institution.nodes.exists():
```

Comment:
Drive-by: What if there are institutional users but no nodes? Probably could break this up into two `if` statements. Or is it even necessary? The query set counts would likely just be 0, and an empty base query set should run probably about as quickly as the `exists()` clauses, or close enough to.

Reply:
That's a good point - It is probably better if every possible institution-user possibility is retrievable, even if 0, from ES than requiring checks to be written in python when trying to retrieve the data, is that correct?

If so, I think removing that logic and allowing for 0 counts to be written is the best course of action.